probe/kubernetes/controls.go

```diff
@@ -78,33 +78,21 @@ func (r *Reporter) CapturePod(f func(xfer.Request, string, string) xfer.Response
 func (r *Reporter) CaptureResource(f func(xfer.Request, string, string, string) xfer.Response) func(xfer.Request) xfer.Response {
```

Comment:
This function should be called `CaptureDeployment` now, by analogy with `CapturePod`.